knative-sandbox-OWNERS_ALIASES

```diff
@@ -99,6 +99,7 @@ aliases:
   - matzew
   eventing-rabbitmq-approvers:
   - benmoss
+  - chunyilyu
```

Comment:
nit: This file is auto-generated from the knative-sandbox.yaml file

Reply:
Thanks! Updated